Integrations/integration-Jask_v2.yml

```diff
@@ -0,0 +1,1540 @@
+description: Freeing the analyst with autonomous decisions
+detaileddescription: |-
+  Integration currently defaults to v1 of the Jask, documentation of this api available at
+  - https://[CUSTOMER].portal.jask.ai/api/v1/docs
+
+  Add in your Jask URL:
+  - https://[CUSTOMER].portal.jask.ai/api
+
+  You can retrieve your personal API key from the following URL under the "Profile" tab:
+  - https://[CUSTOMER].portal.jask.ai/config
+
+  Fetching Insights:
+  Defaults to fetching Insights from the current day (Today), or if you want to go back further you can change the fetch days to the number of days you want to go back.  If you have lots of Insights make sure to change the Fetch Limit as well.
+configuration:
+- display: Jask URL
+  name: URL
+  defaultvalue: https://[CUSTOMER].portal.jask.ai/api
+  type: 0
+  required: true
+- display: Jask API Key
+  name: apiKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Jask API Version
+  name: apiVersion
+  defaultvalue: v1
+  type: 0
+  required: true
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: "true"
```

Comment:
Please remove default value here and move the proxy parameter to be above the fetch ones